packages/react/src/auto/polaris/PolarisAutoTable.tsx

```diff
@@ -65,7 +65,10 @@ export const PolarisAutoTable = <
   props: AutoTableProps<GivenOptions, SchemaT, FinderFunction, Options>
 ) => {
   const { model } = props;
-  const componentKey = `${[model.findMany.namespace, model.findMany.modelApiIdentifier].join("_")}AutoTable`;
+
+  const componentKey = model
```

Comment:
This seems a bit weird to me -- why do we sent the key to undefined here?

Reply:
We throw the error in the hook, which is below this area. So at that moment the `model` is undefined.

Reply:
Hmm then with apologies I think I might change my previous feedback about moving the errors into the hooks -- I think we want to validate the inputs as soon as we can so that we don't have to do things like this.